pkg/kubelet/cm/devicemanager/device_plugin_stub.go

```diff
@@ -104,6 +110,43 @@ func (m *Stub) Stop() error {
 	return m.cleanup()
 }
 
+// GetInfo is the RPC which return pluginInfo
+func (m *Stub) GetInfo(stream watcherapi.Registration_GetInfoServer) error {
+	log.Println("GetInfo")
```

Comment:
Use glog

Reply:
I am following existing pattern in this file, which is to use log. May be in a later PR we can improve this individually.

Reply:
sed -i 's/log.Println/glog.V(5)/'